contracts/.slither.config-pr.json

```diff
@@ -0,0 +1,4 @@
+{
+  "filter_paths": "(openzeppelin|mocks/|test/|tests/|testhelpers)",
+  "detectors_to_exclude": "pragma"
```

Comment:
suggestion: I think it might be useful to have a config file per product so that each one can have custom configs. For CCIP for example, we might want to add additional detectors to exclude such as:
- solc-version
- naming-convention
- assembly
- reentrancy-events
- timestamp
- calls-loop
- unused-return
- ...

And the above might not be applicable to all products.

Reply:
added